Tribler/Core/Modules/restapi/root_endpoint.py

```diff
@@ -41,11 +40,10 @@ def start_endpoints(self):
         on a fully started Tribler.
         """
         child_handler_dict = {"search": SearchEndpoint, "channels": ChannelsEndpoint, "mychannel": MyChannelEndpoint,
-                              "settings": SettingsEndpoint, "variables": VariablesEndpoint,
-                              "downloads": DownloadsEndpoint, "createtorrent": CreateTorrentEndpoint,
-                              "torrents": TorrentsEndpoint, "debug": DebugEndpoint, "shutdown": ShutdownEndpoint,
-                              "multichain": MultichainEndpoint, "statistics": StatisticsEndpoint,
-                              "torrentinfo": TorrentInfoEndpoint}
+                              "settings": SettingsEndpoint, "downloads": DownloadsEndpoint,
```

Comment:
As it is possible to set default values for these ports, I think it would be better to merge these. If you set a default value in the config, you would expect that setting to be retrievable via the `settings` endpoint, right? Moreover, it is then not a variable any more.